lib/transformers/typescript.js

```diff
@@ -1,5 +1,5 @@
 const ensureRequire = require('../ensure-require')
-const babelJest = require('babel-jest')
+const babelJest = require('babel-jest').default
```

Comment:
Has jest changed how it processes modules with default exports? I guess this is the breaking change?